utils/responses.ts

```diff
@@ -11,7 +11,8 @@ export interface IResponseErrorGone extends IResponse<"IResponseErrorGone"> {
  */
 export function ResponseErrorGone(detail: string): IResponseErrorGone {
```

Comment:
we can remove this file and use https://github.com/pagopa/io-ts-commons/blob/master/src/responses.ts#L550 
(I've added it today)

Reply:
done